assets/css/app.scss

```diff
@@ -1,91 +1,5 @@
-/* This file is for your main application css. */
```

Comment:
I feel most of the classes defined in this file will be useful since we are working with LiveView. What I will suggest is to create a new file `liveview.css` and import it in `app.css` (see https://github.com/beamkenya/beamkenya_site/pull/6/files#r579540971). And a new file, called `components.css`, can be created then move the import of tailwind directives into, that way the `app.css` is less bloated.

Reply:
yeah, like for tailwind css, a separate file will work, the tailwind directives can grow with time with much customizations and creating custom components as we grow